### DISCUSSION TITLE
How do I prevent griefing in my mining tunnels?

### DISCUSSION THREAD

**Discussion by shai** [GriefPrevention/GriefPrevention#824] [Q&A] [answered]
I'm not sure if this is even possible .. but I want to be able to mine underground and not get griefed there.. is that possible?

**Comment by RoboMWM** [comment]
What do you mean by this

**Comment by bigpreshkt** [comment, marked as answer]
If the tunnels are within an area you've claimed, the claim will automatically extend downwards to protect them if you simply build something (place a block).  If they're outside of a claim, then they are of course unprotected.

**Comment by bigpreshkt** [comment]
If you mine further out than your claim, top tip: place doors in your mines just before they leave the claim, so that someone that gets into them from outside can't then get in to your claim (same as you'd put doors on your house, etc)

**Comment by shai** [comment]
@bigpreshkt Thanks! That's exactly what I wanted to know. Also, I'm guessing that since I need to "earn" more point blocks, I can't really utilize the strip mining technique, right? Since that technique really extends a lot...

**Comment by RoboMWM** [comment]
Oh that's what you meant, I thought you meant like in general as you dig in an arbitrary area. Context matters, thanks bigpreshkt c:
